### ISSUE TITLE
Victory Bars - Spacing between bars 

### ISSUE THREAD

**Issue by bhaveshbarskar** [FormidableLabs/victory#2002]
Hello Team, Is there a way to control the spacing between the bars in a Victory Bars. So basically I am looking to have a fixed width for my bars and the spacing between the bars should also be fixed, currently I tried with barWidth and barRatio both, so its actually setting the space between the bars based on number of bars. so lets say if I have 10 bars the spacing will auto adjust  in case of 2 bars only it will be huge.

So what I want is the spacing should remains same irrespective of bars, its should be constant along with width of bars. so If i have 10 bars lets say so spacing should be 10px lets say, and also in case when i have only two bars, the spacing should still remains 10px it should not get auto adjust.

**Comment by tgalfin** [comment]
@bhaveshbarskar, can you post a screenshot of what the chart looks like and a drawing of what you want to happen?

**Comment by bhaveshbarskar** [comment]
Hey @tgalfin  

This is how my graph looks when there is less number  of bars. (Spacing b/w the bars is to much)
![image](https://user-images.githubusercontent.com/16028984/138342396-a9e452a9-060e-4675-8df0-3af477749a01.png)


But ideally I want to have less spacing between the bars  like - 
![image](https://user-images.githubusercontent.com/16028984/138342625-d420249b-3eea-4e85-adce-f466f46f6c8c.png)

so basically all I want is for first scenario the spacing should be same as second pic.

**Comment by beccanelson** [comment]
Hi @bhaveshbarskar! We don't currently provide a way to specify particular spacing, probably because it would be difficult to have set bar widths and spacing while still keeping the chart size fully responsive. There are other variables you can play with like `barWidth`, `barRatio`, and `domainPadding` that can be used to adjust the size and spacing of your bar chart.

**Comment by rahul-patil10** [comment]
I have same issue but I use domain={{x: [0, 6]}} and my problem is solved. The space between bars is reduced.
